{"name": "test_try_accept_after_accept", "status": "failed", "statusDetails": {"message": "AssertionError: Expected status code: 304, but actual 404. \n   Responce:\n  \t Status:404 \n  \t Text: User does not exist!\nassert 404 == 304\n  +404\n  -304", "trace": "self = <test_accept_user.TestApiAccept object at 0x000001BAFAFD4700>, get_random_user = {'email': 'xgarcia@yahoo.com', 'password': 'd!7gNq(Zt)', 'username': 'sandersnorman'}\n\n    @allure.epic('Awesome test application')\n    @allure.feature('API tests')\n    @allure.story('Accept user')\n    def test_try_accept_after_accept(self, get_random_user):\n        name = get_random_user[\"username\"]\n        self.api_client.accept_user(name)\n        res = self.api_client.accept_user(name)\n>       assert res.status_code == 304, f\"Expected status code: 304, but actual {res.status_code}. \" \\\n                                       f\"\\n Responce:\\n\\t Status:{res.status_code} \\n\\t Text: {res.text}\"\nE       AssertionError: Expected status code: 304, but actual 404. \nE          Responce:\nE         \t Status:404 \nE         \t Text: User does not exist!\nE       assert 404 == 304\nE         +404\nE         -304\n\ntests\\api_test\\test_accept_user.py:26: AssertionError"}, "start": 1640010793052, "stop": 1640010793102, "uuid": "e96ae207-a7d5-4943-a7ce-58db8b63b209", "historyId": "1deadff667e9505e3a7f6e7ca9ef47b2", "testCaseId": "62f13c072c38942c3b08fb251a6231a6", "fullName": "tests.api_test.test_accept_user.TestApiAccept#test_try_accept_after_accept", "labels": [{"name": "feature", "value": "API tests"}, {"name": "epic", "value": "Awesome test application"}, {"name": "story", "value": "Accept user"}, {"name": "parentSuite", "value": "tests.api_test"}, {"name": "suite", "value": "test_accept_user"}, {"name": "subSuite", "value": "TestApiAccept"}, {"name": "host", "value": "DESKTOP-LERA"}, {"name": "thread", "value": "6588-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_test.test_accept_user"}]}